web-client/src/pages/admin/AdminInventoryPage.tsx

```diff
@@ -1,5 +1,100 @@
-import UnderConstruction from "@/components/UnderConstruction";
+import { Button } from "@/components/ui/button";
+import { Search } from "lucide-react";
+import { ChevronDown } from "lucide-react";
+import { DropdownMenu, DropdownMenuItem, DropdownMenuTrigger, DropdownMenuContent } from "@/components/ui/dropdown-menu";
+import InventoryTable from '@/components/InventoryTable';
+import { useNavigate } from 'react-router-dom';
+import { useGetSocks } from '@/api/inventory/queries';
+import { useState } from 'react';
 
 export default function AdminInventoryPage() {
-  return <UnderConstruction pageName="Inventory" />;
+  const navigate = useNavigate();
+
+  const [page, setPage] = useState(1);
+  const limit = 8;
+  const offset = (page - 1) * limit;
+
+  const { data, error, isLoading } = useGetSocks(limit, offset);
+
+  if (isLoading) {
+    return <div>Loading...</div>;
```

Comment:
If you have time, experiment and make a custom faily barebones loading skeleton for the table space